build/configs/rtl8721csm/hello/defconfig

```diff
@@ -808,12 +808,12 @@ CONFIG_TLS_MPI_MAX_SIZE=512
 #
 # HW Options
 #
-CONFIG_TLS_HW_RNG=y
-CONFIG_TLS_HW_DH_PARAM=y
-CONFIG_TLS_HW_ECDH_PARAM=y
-CONFIG_TLS_HW_RSA_VERIFICATION=y
-CONFIG_TLS_HW_ECDSA_VERIFICATION=y
-CONFIG_TLS_HW_RSA_ENC=y
+#CONFIG_TLS_HW_RNG=y
```

Comment:
Please do not edit the defconfig manually.
You can modify the defconfig through menuconfig.

Reply:
sure can